Build fails because openjdk-8-jre is not found

```
$mvn install
...
[ERROR] Failed to execute goal com.spotify:docker-maven-plugin:0.4.13:build (build) on project openjdk8: Exception caught: The command '/bin/sh -c echo 'deb http://httpredir.debian.org/debian jessie-backports main' > /etc/apt/sources.list.d/jessie-backports.list  && apt-get -q update  && apt-get -y -q --no-install-recommends install     ca-certificates     openjdk-8-jre=8u102'-*'     netbase     wget     unzip  && apt-get clean  && rm /var/lib/apt/lists/*_*' returned a non-zero code: 100 -> [Help 1]
```

More detailed:

```
$/bin/sh -c echo 'deb http://httpredir.debian.org/debian jessie-backports main' > /etc/apt/sources.list.d/jessie-backports.list  && apt-get -q update  && apt-get -y -q --no-install-recommends install     ca-certificates     openjdk-8-jre=8u102'-*'     netbase     wget     unzip  && apt-get clean  && rm /var/lib/apt/lists/*_*

Get:1 http://security.debian.org jessie/updates InRelease [63.1 kB]
Ign http://httpredir.debian.org jessie InRelease
Get:2 http://httpredir.debian.org jessie-updates InRelease [145 kB]
Get:3 http://httpredir.debian.org jessie Release.gpg [2373 B]
Get:4 http://security.debian.org jessie/updates/main amd64 Packages [407 kB]
Get:5 http://httpredir.debian.org jessie Release [148 kB]
Get:6 http://httpredir.debian.org jessie-updates/main amd64 Packages [17.6 kB]
Get:7 http://httpredir.debian.org jessie/main amd64 Packages [9064 kB]
Fetched 9847 kB in 2s (3552 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
E: Unable to locate package openjdk-8-jre
```

--- a/pom.xml
+++ b/pom.xml
@@ -34,7 +34,7 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <maven.build.timestamp.format>yyyy-MM-dd_HH_mm</maven.build.timestamp.format>
     <appengine.api.version>1.9.40</appengine.api.version>
-    <openjdk8.version>8u102</openjdk8.version>
+    <openjdk8.version>8u111</openjdk8.version>
     <docker.tag.prefix></docker.tag.prefix>
     <docker.appengine.image>gcr.io/google_appengine/base</docker.appengine.image>
   </properties>

